galsim/chromatic.py

```diff
@@ -643,18 +654,17 @@ def rotate(self, theta):
 
         @returns the rotated object.
         """
-        import math
         if hasattr(theta, '__call__'):
             def buildRMatrix(w):
                 sth, cth = theta(w).sincos()
                 R = np.array([[cth, -sth],
-                              [sth,  cth]], dtype=float)
+                              [sth, cth]], dtype=float)
```

Comment:
This one is ridiculous.  We used to have a nicely aligned matrix, but you changed it to be unaligned to satisfy some one-space-only rule.  We have to ignore that one and revert this change.